skimage/segmentation/_clear_border.py

```diff
@@ -21,7 +19,7 @@ def clear_border(labels, buffer_size=0, bgval=0, in_place=False):
 
     Returns
     -------
-    labels : (N, M) array
+    labels : (M[, N[, ..., P]]) array
         Cleared binary image.
```

Comment:
As above - I simply maintained the previous docstrings in those respects, but if your suggested changes are preferred I can certainly include them in this pull request... or would it make more sense to do that in a different pull request?